src/Allegro.CosmosDb.BatchUtilities/Allegro.CosmosDb.BatchUtilities/Allegro.CosmosDb.BatchUtilities.csproj

```diff
@@ -3,25 +3,28 @@
   <PropertyGroup>
     <PackageId>Allegro.CosmosDb.BatchUtilities</PackageId>
     <PackageDescription>Utilities for performing batch operations in Azure CosmosDb, such as rate limiting and autoscaling.</PackageDescription>
-    <Version>1.0.1</Version>
+    <Version>1.0.2</Version>
   </PropertyGroup>
 
-  <Import Project="..\..\Package.Build.props"/>
+  <Import Project="..\..\Package.Build.props" />
 
   <ItemGroup>
     <PackageReference Include="AsyncFixer" Version="1.5.1">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Asyncify" Version="0.9.7"/>
+    <PackageReference Include="Asyncify" Version="0.9.7">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
     <PackageReference Include="Meziantou.Analyzer" Version="1.0.676">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.22.0"/>
-    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="5.0.0"/>
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="5.0.0"/>
-    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="5.0.0"/>
+    <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.22.0" />
```

Comment:
Could we update to the newest version? According to the release notes [here](https://github.com/Azure/azure-cosmos-dotnet-v3/releases) there are some fixes (e.g. differentiating between SDK and backend generated 503s) and perf improvements. I'll create a PR in our internal repo and we can test it on one service.

Reply:
I think these open-source utilities should require the minimal supported SDK version. Package users can update to higher versions in their projects, but I would not force them.